projects/ui/src/lib/components/po-field/po-combo/po-combo-base.component.ts

```diff
@@ -292,17 +299,37 @@ export abstract class PoComboBaseComponent implements ControlValueAccessor, OnIn
    */
   @Input('p-icon') icon?: string;
 
+  /** Indica que a lista definida na propriedade p-options será ordenada pela descrição. */
+  @Input('p-sort') set sort(sort: boolean) {
+    this._sort = <any>sort === '' ? true : convertToBoolean(sort);
```

Comment:
1- Ao alterar a propriedade `sort` dinamicamente, por exemplo no sample labs, não está mais reordenando os itens.

2- A função convertToBoolean já trata valores vazios, poderia ser apenas: `this._sort = convertToBoolean(sort);`

Reply:
1 - O bind da propriedade `sort` depende da inclusão de mais dados. Isso já é o comportamento atual do sample labs e o mantive assim.

2 - ok refatorado :)